internal/bind/params.go

```diff
@@ -21,7 +21,6 @@ var (
 	errMultipleQueryParameters = errors.New("only one query arg *table.QueryParameters allowed")
 )
 
-//nolint:gocyclo
 func toValue(v interface{}) (_ types.Value, err error) {
```

Comment:
no need to split the function.

in new style it hase same big type switch, but distributed for few functions with double dependencies (switch case `[]byte, *[]byte, string, *string` must be equal to cases in  `convertBytesAndStrings` it if difficult to read, test, change), it is more difficult, then one big type switch.